### ISSUE TITLE
Old CMS: In Places, missing bilingual accessibility note

### ISSUE THREAD

**Issue by saumier** [culturecreates/footlight-app#262]
Dependant on https://github.com/culturecreates/footlight-calendar-api/issues/408
